java/code/src/com/suse/manager/webui/services/impl/SaltService.java

```diff
@@ -174,6 +175,12 @@ public enum KeyStatus {
         ACCEPTED, DENIED, UNACCEPTED, REJECTED
     }
 
+    private static final String SENSITIVE_DATA_REGEX =
+            "-----BEGIN CERTIFICATE-----" +
+            "|" +
+            "-----BEGIN RSA PRIVATE KEY-----";
```

Comment:
Don't you match the data to the end of the certificate?

Reply:
yeah it was my initial approach, but then I though we could make the regex pattern a lot simpler jut pointing to the beginning of the CA. Do you think we still need to match it entirely to the end?

Reply:
yes... the security-sensitive thing is between the BEGIN and END marks